src/view/base-template.twig

```diff
@@ -93,7 +93,7 @@
     </div>
     {% endif %}
   </header>
-  <main id="main-container" class="{% if request.vocabid == '' or global_search %} frontpage{% elseif concept_uri != '' %} termpage{% elseif request.vocab != '' %} vocabpage{% if list_style %} {{ list_style }}{% endif %}{% endif %}">
+  <main id="main-container" class="{{pageType}}{% if list_style %} {{ list_style }}{% endif %}">
```

Comment:
This gets rid of the old and IMHO misleading page type classes `frontpage`, `termpage` and `vocabpage` , which is great! :tada: 

However, there are still many occurrences of these classes in `skosmos.css` that should be changed or removed.